### PR TITLE
Add teaser on textblock

### PR DESCRIPTION
**Pull request by shylux**
Extranet: https://extranet.4teamwork.ch/extern/0202-projekt-edubs-ict-basler-schulen/tracker-edubs/603

@maethu

### CODE REVIEW

**Comment by shylux** [comment]
@maethu Depends on https://github.com/4teamwork/simplelayout.ui.base/pull/11

Changes in new commit. Had to rebase - tested upgrade and hope i didn't break something else.

File: ftw/contentpage/profiles/default/jsregistry.xml [hunk 1]
```diff
@@ -8,4 +8,9 @@
 
  <javascript cacheable="True" compression="safe" cookable="True"
     enabled="True" expression="" id="++resource++ftw.contentpage.resources/toggle_faq.js" inline="False"/>
+
+ <javascript cacheable="True" compression="safe" cookable="True"
```
Comment (maethu) [inline ftw/contentpage/profiles/default/jsregistry.xml]: @shylux Please do not register this js globaly.  It's only used in very specific usecases (textBlock + right permission).
You can put it onto a AT filed as helper js.

**Comment by maethu** [comment]
@phgross Technically it looks good :+1: please check the JS part I mentioned.

File: ftw/contentpage/profiles/default/jsregistry.xml (hunk 1 above)
Comment (jone) [inline ftw/contentpage/profiles/default/jsregistry.xml]: I've discussed this. It seems that helper_js does not work with resources; the JS must be in a skins directory. :confused:

**Comment by mbaechtold** [comment]
@jone, should the version in ```profiles/default/metadata.xml``` be incremented too? I'm trying to integrate this feature into a project and *ftw.upgrade* is complaining:

> This upgrade's target version is higher than the profile's filesystem version. Please update the corresponding metadata.xml!

**Comment by maethu** [comment]
@mbaechtold Goot point - sure it should. You can fix it by directly push the right version into the master.

**Comment by mbaechtold** [comment]
@maethu, this is done in bbb555cb89abe12a04671a97c72db30c101113c6.

**Comment by maethu** [comment]
@mbaechtold thx!

**Comment by jone** [comment]
@mbaechtold thanks, didn't see that :confused:

**Pull request merged**